warehouse/templates/pages/help.html

```diff
@@ -158,6 +160,20 @@ <h3 id="verified-email">{{ verified_email() }}</h3>
         <p>
           You can manage your account's email addresses in your <a href="{{ request.route_path('manage.account') }}">Profile</a>. This also allows for sending a new confirmation email for users who signed up in the past, before we began enforcing this policy.
         </p>
+
+
+        <h3 id="compromised-password">{{ compromised_password() }}</h3>
+          <p>
+            PyPI securely checks if passwords users supply when registering, authenticating, or updating their password have appeared in known data breaches.
+          <p>
+            During each of these processes, PyPI generates a SHA-1 hash of the supplied password and uses the first five (5) characters of the hash to check the <a href="https://haveibeenpwned.com/API/v2#SearchingPwnedPasswordsByRange" target="_blank" rel="noopener">Have I Been Pwned API</a> and determine if the password has been previously compromised. The plaintext password is never stored by PyPI or submitted to the Have I Been Pwned API.
+          </p>
+          <p>
+            PyPI will not allow such passwords to be used when setting a password at registration or updating your password.
+          </p>
+          <p>
+            If the password you supplied is noted as having appeared in known compromises, you should change it all other places that you use it as soon as possible.
```

Comment:
I suggest we include the text of the error message here:

>  This password has appeared in a breach or has otherwise been compromised and cannot be used.

so people will find this page when searching the web for the error message, and will find this answer when using Control-F on the page.

Reply:
I'm not sure of a good way to incorporate the full error message, but whenever it is shown, it displays a link to this FAQ entry. Is that sufficient?

Reply:
It's a good step and it helps, but I'm sorry to say no, I don't think it's sufficient, because it doesn't help people who do a control-F. Here's a suggested phrasing:

> PyPI will not allow such passwords to be used when setting a password at registration or updating your password. 
> If you receive an error message saying that "This password has appeared in a breach or has otherwise been compromised and cannot be used", you should change it all other places that you use it as soon as possible.